website/docs/Installation/creating-a-podman-machine-with-podman-desktop.md

```diff
@@ -0,0 +1,46 @@
+---
+sidebar_position: 11
+title: Custom Podman machine
+description: Podman Desktop can assist you to install custom Podman machine on Windows and macOS.
+tags: [podman-desktop, podman, installing, windows, macOS]
+keywords: [podman desktop, containers, podman, installing, installation, windows, macos, podman machine]
+---
+
+# Creating a Podman machine with Podman Desktop
+
+On macOS and Windows, running the Podman container engine requires running a Linux distribution on a virtual machine.
+
+By default, Podman Desktop is initializing a Podman machine with a standard configuration.
+
+Consider creating a custom Podman machine to:
+
+- Control the assigned resources: CPUs, memory, and disk size.
+- Use a custom boot image.
+- Use the rootful connection by default, for example to run Kind.
+
+#### Prerequisites
+
+- The Podman executable is installed.
+
+#### Procedure
+
+1. Go to **Settings > Resources**.
+1. In the **Podman** tile, click **Create new**.
+1. In the **Create a Podman machine** screen:
+
+   1. **Name**:
+      Enter a name, such as `podman-machine-default`.
+   1. **CPU(s)**:
+      Select the number of CPUs.
+   1. **Memory**:
+      Select the memory size.
+   1. **Disk size**:
+      Select the disk size.
+   1. **Image path (Optional)**:
+      Select a bootable image containing a virtual machine with Podman.
+   1. **Machine with root privileges**:
+      Enable to use the rootful connection by default.
```

Comment:
weve gotten question with regards to rootful / root privileges and why we need to enable them, I think we should link this section to an article / explain in more detail.

Reply:
This is explained in the prerequisite section.